cmd/retry.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"spot-oceancd-cli/pkg/oceancd"
+)
+
+// retryCmd represents the retry command
+var (
+	retryDescription      = "Available for the last rolled back SpotDeployment only. With this action you will be able to retry your full rollout"
+	retryShortDescription = "Retry a rollout"
+	retryExample          = fmt.Sprintf(rolloutActionExampleTemplate,
```

Comment:
Add a method in rollout cmd that get the description and action and return the example

Reply:
ok